browser/ai_chat/ai_chat_settings_helper.cc

```diff
@@ -81,6 +81,14 @@ void AIChatSettingsHelper::GetModelsWithSubtitles(
       modelWithSubtitle.subtitle =
           l10n_util::GetStringUTF8(IDS_CHAT_UI_CHAT_CLAUDE_INSTANT_SUBTITLE);
       is_key_handled = true;
+    } else if (modelWithSubtitle.model->key == "chat-claude-haiku") {
+      modelWithSubtitle.subtitle =
+          l10n_util::GetStringUTF8(IDS_CHAT_UI_CHAT_CLAUDE_HAIKU_SUBTITLE);
+      is_key_handled = true;
+    } else if (modelWithSubtitle.model->key == "chat-claude-sonnet") {
+      modelWithSubtitle.subtitle =
+          l10n_util::GetStringUTF8(IDS_CHAT_UI_CHAT_CLAUDE_SONNET_SUBTITLE);
+      is_key_handled = true;
```

Comment:
cc for @Brandon-T as I don't think this class / function with the model descriptions is shared with iOS yet, unfortunately.

Reply:
Yes. iOS has to add these manually. We don't use the WebUI settings screen. I might be able to refactor this in the future to pull in the info and they'd both share the underlying code.

But for now, iOS has to add this.

Reply:
Should I add it to iOS or open an issue for a followup?